app/controllers/calculator_constants_controller.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+class CalculatorConstantsController < ApplicationController
+  def index
+    @calculator_constants = CalculatorConstant.all
+  end
+
+  def update
+    updated_fields = []
+    submitted_constants = params['calculator_constants'][0]
+    update_calculator_constant(submitted_constants, updated_fields)
+    unless updated_fields.empty?
+      flash[:success] = {
+        updated_fields: updated_fields
+      }
+    end
+    redirect_to action: :index
+  end
+
+  private
+
+  def update_calculator_constant(submitted_constants, updated_fields)
+    constant_fields = CalculatorConstant.where(name: submitted_constants.keys)
+    constant_fields.each do |constant|
+      submitted_value = submitted_constants[constant.name]
```

Comment:
There's an assumption that the params hash key represents the `CalculatorConstant.name`. This assumption isn't very readable and could start breaking down if more attributes are added to the `CalculatorConstant` model.

I think a more standard rails practice & restful design would be to lay out each attribute name explicitly:

```
"calculator_constants" => [
  {name: "foo", float_value: 999},
  {name: "bar", float_value: 888}
]
```

This also has the added benefit of allowing you to easily use rails `strong_params` if that's something you're interested in.

Reply:
I'm going to look into this further and try to address it in a separate PR.